### PR TITLE
[#2PlaysAMonth]: Cricket Game 

### PR DESCRIPTION
**Pull request by SamirMishra27**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

In this React play I'm going to make and demonstrate a Web Cricket game using React.js.
Play Cricket and Bat against the computer to chase down the given target of runs with few overs and wickets in hand. You will level up if you successfully chase the target or else you will lose!
Hit the ball carefully when it comes to you! 🏏

Implemented and submitted as per Issue: #951 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Project was manually tested both on my local system and mobile device to ensure all features are working as well as responsiveness.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

# Screenshots or example output
Web capture_15-2-2023_233143_localhost

Screenshot 2023-02-16 002305

Screenshot 2023-02-14 070325

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "add": "^2.0.6",
     "axios": "^0.27.2",
     "browser-image-compression": "^2.0.0",
+    "classnames": "^2.3.2",
```
Comment (atapas) [inline package.json:20]: Why do you need it?

File: src/plays/cricket-game/Readme.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- Language: js
+- Level: Intermediate
+
+## Creator Information
+
+- User: SamirMishra27
+- Gihub Link: https://github.com/SamirMishra27
+- Blog: 
+- Video:
```
Comment (atapas) [inline src/plays/cricket-game/Readme.md:15]: Have you recorded the video stream?

File: package.json (hunk 1 above)
Comment (SamirMishra27) [reply package.json:20]: I'm using `classnames` module in few components to break down the `className` string into multiple lines and organize it better. For example in `src/components/Pitch.js` I've used the module in 2 elements to separate its variables and styles.

If this module's usage is unnecessary, I will remove it.

File: src/plays/cricket-game/Readme.md (hunk 2 above)
Comment (SamirMishra27) [reply src/plays/cricket-game/Readme.md:15]: I have recorded a video of me demonstrating the project in motion. 
I was unable to record or stream myself coding this project live.

File: src/plays/cricket-game/styles.css [hunk 3]
```diff
@@ -0,0 +1,289 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400;500;600;700;800;900&family=Signika+Negative:wght@300;400;500;600;700&display=swap');
+
+/* enter styles here */
+
```
Comment (atapas) [inline src/plays/cricket-game/styles.css:4]: You gave a class name called `cricket-home-body` already. Please use it to prefix all the class names below.. like `.cricket-home-body img`, `.cricket-home-body .modal`, etc. it is applicable to all the class names in this file.

File: src/plays/cricket-game/styles.css [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+* {
```
Comment (atapas) [inline src/plays/cricket-game/styles.css:5]: No * override is allowed. Use the main class name `.cricket-home-body`

**Review by atapas — CHANGES_REQUESTED** [review]

**Comment by atapas** [comment]
@SamirMishra27 Please create the demo recording and add the link by editing your play.

File: src/plays/cricket-game/styles.css (hunk 3 above)
Comment (SamirMishra27) [reply src/plays/cricket-game/styles.css:4]: Fixed!
https://github.com/reactplay/react-play/pull/972/commits/c7dba9eedf67f7176c2f523e63b23b6d6720673e

File: src/plays/cricket-game/styles.css (hunk 4 above)
Comment (SamirMishra27) [reply src/plays/cricket-game/styles.css]: Fixed!
https://github.com/reactplay/react-play/pull/972/commits/c7dba9eedf67f7176c2f523e63b23b6d6720673e

File: src/plays/cricket-game/Readme.md (hunk 2 above)
Comment (SamirMishra27) [reply src/plays/cricket-game/Readme.md]: Demo recording has been added in the `Readme.md` file of react-play.
https://github.com/reactplay/react-play/pull/972/commits/c7dba9eedf67f7176c2f523e63b23b6d6720673e

File: src/plays/cricket-game/styles.css [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+    box-sizing: border-box;
+}
+
+.cricket-home-body img {
+    pointer-events: none;
+    user-select: none;
+    cursor: none;
+}
+
+.cricket-home-body * {
```
Comment (atapas) [inline src/plays/cricket-game/styles.css:24]: You do not need to give * here... just move the font style to the '.cricket-home-body' declaration

**Review by atapas — CHANGES_REQUESTED** [review]

File: src/plays/cricket-game/styles.css (hunk 5 above)
Comment (SamirMishra27) [reply src/plays/cricket-game/styles.css:24]: @atapas I already tried putting 
```css
.cricket-home-body {
    font-family: "Signika Negative", Arial, Helvetica, sans-serif;
}
```
Earlier... It did not work, the font only applied to that particular class but the child elements didn't got that font.

Comment (SamirMishra27) [reply src/plays/cricket-game/styles.css]: Kindly ignore my earlier comment.
I've resolved the issue in https://github.com/reactplay/react-play/pull/972/commits/d2ef717fad6d97d7a4129d67e1c79563bb4d5ef2

**Comment by atapas** [comment]
@SamirMishra27 Please record your demo and add the link by editing your play

**Review by atapas — APPROVED** [review]

**Pull request merged**